pages/index.vue

```diff
@@ -0,0 +1,519 @@
+                    data-aos-delay="50">
+
+                <img src="/hero-img-2.png" alt=" hero img-1" class="lg:-mr-10 lg:z-20" data-aos="zoom-in"
+                    data-aos-delay="100">
+
+                <img src="/hero-img-3.png" alt=" hero img-1" class="lg:-mr-16 lg:mt-10 lg:z-10" data-aos="zoom-in"
+                    data-aos-delay="150">
+
+                <img src="/hero-img-4.png" alt=" hero img-1" data-aos="zoom-in" data-aos-delay="200">
+            </div>
+        </div>
+    </section>
+
+    <!-- project section -->
+    <section class="mt-40 lg:mt-[188px]">
+        <div class="container relative" data-aos="fade-up">
+            <p class="text-white text-3xl md:text-6xl">
+                Il Padel World Club DAO <br>
+                mira a
+                <span class="inline-flex items-center gap-1 bg-secondary text-app-bg px-2 my-1">
+                    rivoluzionare
+                    <icon-rotate />
+                </span> <br>
+
+                l'esperienza del
+                <span class="inline-flex items-center gap-1 bg-secondary text-app-bg px-2 my-1">
+                    padel
+                    <icon-livello />
+                </span>
+                creando <br>
+                un circuito elitario di club <br>
+                diffusi <span class="inline-flex items-center gap-1 bg-secondary text-app-bg  px-2 my-1">
+                    globalmente
+                    <icon-globe />
+                </span> attraverso <br>
+                la tecnologia web3.
+            </p>
+            <span
+                class=" text-app-bg bg-[#D3E2E8] px-2 md:px-4 py-2 text-xl md:text-3xl absolute  -top-20 left-6 md:-left-2 rotate-[-5.45deg]">IL
+                PROGETTO</span>
+        </div>
+    </section>
+
+    <!-- marque section -->
+    <section class="mt-40 lg:mt-[131px] pt-11 pb-16 border border-white border-l-0 border-r-0 border-opacity-25">
+        <vue3-marquee :clone="true" :duration="20" :direction="'reverse'" :pause-on-hover="true"
+            @onOverflowCleared="alert()" class=" overflow-y-hidden">
+            <div class="text-white font-medium text-4xl md:text-6xl lg:text-[86px] flex items-center gap-5 italic hover:text-[#4D696D] hover:not-italic mr-10 transition duration-200 ease-linear"
+                v-for="text in marqueText" :key="text">
+                {{ text }}
+                <icon-arrow-right-large class="w-12 h-12 md:w-24 md:h-24 lg:w-[91px] lg:h-[84px]" />
+            </div>
+        </vue3-marquee>
+    </section>
+
+    <!-- partner section -->
+    <section class="mt-[156px]">
+        <div class="container ">
+            <div class="bg-[#1A3E43] relative max-w-[1254px] mx-auto py-8" data-aos="fade-up">
+                <div
+                    class="text-4xl lg:text-3xl  xl:text-5xl lg:!leading-[56px] pt-[92px] pb-[61px] pl-4 md:pl-[53px] text-[#99A9AB] relative">
```

Comment:
this color `#99A9AB` is used multiple times, so you should add it in the tailwindcss config, is not nice to add here inline also the color everytime.